systemtests/src/main/java/io/enmasse/systemtest/executor/Executor.java

```diff
@@ -26,14 +26,21 @@
     private StreamGobbler stdOutReader;
     private StreamGobbler stdErrReader;
     private Path logPath;
+    private boolean appendLineSeparator;
 
     public Executor() {
+        this(true);
```

Comment:
I don't like calling another constructor, better from my POV is setting appendLineSeparator manually to true